src/client/client4.js

```diff
@@ -312,6 +312,18 @@ export default class Client4 {
         );
     };
 
+    // Sends an api request to get user analyics for:
+    //     1. a user (userId)
+    //     2. a team (teamId)
+    //     3. a specific set of analytics data (name)
+    // Uses getUserRoute to generate 'users/{userId}' for the request url
```

Comment:
This is a classic simple function the function comment can be as succinct as:
```
/**
 * Get the 'name' UserAnalytics data for the specified user in the specified team
 * from the mattermost-server.
 */
```

or if you want
```
/**
 * Get the 'name' UserAnalytics data for the specified user in the specified team
 * from the mattermost-server. UserAnalytics are based on interactions the
 * user had in a particular team.
 *
 * @param {string} userId - The target user for the requested analytics
 * @param {string} teamId - The target team for the requested analytics
 * @param {string} name - The name of the type of analytic data to return
 *
 * @returns {Promise<object>} containing the particular analytics requested
 */
```

Reply:
Altered, used your suggestion.